app/services/finance/npq/calculation_overview_orchestrator.rb

```diff
@@ -23,6 +23,7 @@ def call(event_type:)
             event_type: event_type,
             cpd_lead_provider: cpd_lead_provider,
             contract: contract,
+            interval: interval,
           )
```

Comment:
Do the tests still pass without being updated?

Reply:
Yes, we therefore have some lack of coverage now. I'm not fully certain this will even fix the issue as we haven't yet a mechanism to seed a load of test data to play with